scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -568,6 +593,47 @@ object Config {
     md5Hex(bytes)
   }
 
+  /**
+   * Add a file to be localized to the config. Intended to be used by user code.
+   *
+   * @param qualifiedURI The qualified uri of the cache to be localized
+   * @param config Config to add the cache to
+   *
+   * @return new Config with cached files
+   *
+   * @see basic logic from [[org.apache.hadoop.mapreduce.filecache.DistributedCache.addCacheFile]]
+   */
+  private def addDistributedCacheFile(qualifiedURI: URI, config: Config): Config = {
+    val newFile = DistributedCacheFile
+      .symlinkedUriFor(qualifiedURI)
+      .toString
+
+    val newFiles = config
+      .get(MRJobConfig.CACHE_FILES)
```

Comment:
should we use both the old `map-red` and `mapreduce` keys (if they are different)? Maybe we don't need to.